config/onlstm.conf

```diff
@@ -0,0 +1,27 @@
+//include defaults config
+include "defaults.conf"
+
```

Comment:
Add a brief comment explaining this. If these values are from the Shen paper, say so, and note any differences/exceptions.

Reply:
Added explanations to some in defaults file. The only notable point of difference was learning rate. Also, this is mostly based on only WSJ LM, not on the other tasks, because the optimizer params vary significantly.